addons/web_editor/static/src/js/backend/fields.js

```diff
@@ -83,12 +83,45 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend(TranslatableFieldMi
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * Returns the domain for attachments used in media dialog.
+     * We look for attachments related to the current document. If there is a value for the model
+     * field, it is used to search attachments, and the attachments from the current document are
+     * filtered to display only user-created documents.
+     * In the case of a wizard such as mail, we have the documents uploaded and those of the model
+     *
+     * @private
+     * @returns {Array} "ir.attachment" odoo domain.
+     */
+    _getAttachmentsDomain: function () {
+        var domain = ['|', ['id', 'in', _.pluck(this.attachments, 'id')]];
+        if (this.recordData.model) {
+            domain = domain.concat([
+                '|',
+                '&',
+                '&',
+                ['res_model', '=', this.model],
+                ['res_id', '=', this.res_id|0],
+                ['create_uid', '=', session.uid],
+                '&',
+                ['res_model', '=', this.recordData.model],
+                ['res_id', '=', this.recordData.res_id|0]]);
+        } else {
+            domain = domain.concat([
+                '&',
+                ['res_model', '=', this.model],
+                ['res_id', '=', this.res_id|0]]);
+        }
+        return domain;
+    },
     /**
      * @private
      * @returns {Object} the summernote configuration
      */
     _getSummernoteConfig: function () {
         var summernoteConfig = {
+            model: this.model,
+            id: this.res_id,
```

Comment:
Should we not name this `res_id` too to be consistent ?

Reply:
No because for the editor, this information is already used with id and not res_id